docs/conceptual/l2-cache.rst

```diff
@@ -0,0 +1,763 @@
+
+   * - Remote Write and Atomic Traffic
+
+     - The percent of read requests generated by the L2 cache that are routed to
+       any memory location other than the accelerator's local high-bandwidth
+       memory (HBM) -- for example, the CPU's DRAM or a remote accelerator's
+       HBM. This breakdown does not consider the *size* of the request (meaning
+       that 32B and 64B requests are both counted as a single request), so this
+       metric only *approximates* the percent of the L2-Fabric Read bandwidth
+       directed to a remote location. Note that on current CDNA
+       accelerators, such as the :ref:`MI2XX <mixxx-note>`, requests are only
+       considered *atomic* by Infinity Fabric if they are targeted at
+       :ref:`fine-grained memory <memory-type>` allocations or
+       :ref:`uncached memory <memory-type>` allocations.
+
+     - Percent
+
+   * - Atomic Traffic
+
+     - The percent of write requests generated by the L2 cache that are atomic
+       requests to *any* memory location. This breakdown does not consider the
+       *size* of the request (meaning that 32B and 64B requests are both counted
+       as a single request), so this metric only *approximates* the percent of
+       the L2-Fabric Read bandwidth directed to a remote location. Note that on
+       current CDNA accelerators, such as the :ref:`MI2XX <mixxx-note>`,
+       requests are only considered *atomic* by Infinity Fabric if they are
+       targeted at :ref:`fine-grained memory <memory-type>` allocations or
+       :ref:`uncached memory <memory-type>` allocations.
+
+     - Percent
+
+   * - Uncached Write and Atomic Traffic
+
+     - The percent of write and atomic requests generated by the L2 cache that
+       are targeting :ref:`uncached memory allocations <memory-type>`. This
+       breakdown does not consider the *size* of the request (meaning that 32B
+       and 64B requests are both counted as a single request), so this metric
+       only *approximates* the percent of the L2-Fabric read bandwidth directed
+       to uncached memory allocations.
+
+     - Percent
+
+   * - Read Latency
+
+     - The time-averaged number of cycles read requests spent in Infinity Fabric
+       before data was returned to the L2.
+
+     - Cycles
+
+   * - Write Latency
+
+     - The time-averaged number of cycles write requests spent in Infinity
+       Fabric before a completion acknowledgement was returned to the L2.
+
+     - Cycles
+
+   * - Atomic Latency
+
+     - The time-averaged number of cycles atomic requests spent in Infinity
+       Fabric before a completion acknowledgement (atomic without return value)
+       or data (atomic with return value) was returned to the L2.
+
+     - Cycles
+
+   * - Read Stall
+
+     - The ratio of the total number of cycles the L2-Fabric interface was
+       stalled on a read request to any destination (local HBM, remote PCIe
+       connected accelerator or CPU, or remote Infinity Fabric connected
```

Comment:
Likely should have the (r) on PCIe, as it's the first use.  Again see note about (tm) for instinct

Reply:
done